source/retryable-writes/retryable-writes.rst

```diff
@@ -66,12 +66,40 @@ ClientSession
    name of this object MAY vary across drivers.
 
 Retryable Error
-   Any network exception or a write command response with an error indicating
-   that the node is no longer a primary (e.g. "not master" or "node is
-   recovering" errors). Such errors correspond to those discussed in the SDAM
-   spec's section on `Error Handling`_.
-
-   .. _Error Handling: ../server-discovery-and-monitoring/server-discovery-and-monitoring.rst#error-handling
+   A network exception or write command response error that is possibly transient
+   or indicates that the node is no longer a primary.
+   The following table is a list of error codes of write command responses that
+   MUST be retried:
+
+   .. list-table::
+     :header-rows: 1
+
+     * - Error Name
+       - Error Code
+     * - InterruptedAtShutdown
+       - 11600
+     * - InterruptedDueToReplStateChange
+       - 11602
+     * - NotMaster
+       - 10107
+     * - NotMasterNoSlaveOk
+       - 13435
+     * - NotMasterOrSecondary
+       - 13436
+     * - PrimarySteppedDown
+       - 189
+     * - ShutdownInProgress
+       - 91
+     * - WriteConcernFailed
+       - 64
+     * - HostNotFound
+       - 7
+     * - HostUnreachable
+       - 6
+     * - NetworkTimeout
+       - 89
+     * - SocketException
+       - 9001
```

Comment:
Throw into "Rationale" an explanation. Some of these indicate a node is changing state, or going down, so we want to try resuming. The driver will re-select a server that now satisfies the change stream's read preference. The last four codes are when mongos can't reach a shard, we should resume after those, too - the next time we ask mongos to begin a change stream, mongos may succeed.